js/index.js

```diff
@@ -40,3 +40,37 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+let navA = document.querySelectorAll("nav a");
+//console.log(navA);
+
+// navA.forEach(function(currentValue){
+//   return currentValue.textContent = siteContent.nav[`nav-item-1`];  
+    
+// });
+
+navA[0].textContent = "Services";
+navA[1].textContent = "Product";
+navA[2].textContent = "Vision";
+navA[3].textContent = "Features";
+navA[4].textContent = "About";
+navA[5].textContent = "Contact";
+
```

Comment:
instead of using a string to assigning the `textContent` values you can access the values of the `siteContent` object.
ex.) `navA[0].textContent = siteContent['nav']['nav-item-1']`